packages/mobile/src/app/actions.ts

```diff
@@ -34,6 +34,8 @@ export enum Actions {
   APP_UNMOUNTED = 'APP/APP_UNMOUNTED',
   VERIFICATION_MIGRATION_RAN = 'APP/VERIFICATION_MIGRATION_RAN',
   ANDROID_MOBILE_SERVICES_AVAILABILITY_CHECKED = 'APP/ANDROID_MOBILE_SERVICES_AVAILABILITY_CHECKED',
+  SET_OTA_TRANSLATIONS_LAST_UPDATE = 'APP/SET_OTA_TRANSLATIONS_LAST_UPDATE',
```

Comment:
It's [recommended to model actions as events instead of setters](https://redux.js.org/style-guide/style-guide#model-actions-as-events-not-setters).

```suggestion
  OTA_TRANSLATIONS_UPDATED = 'APP/OTA_TRANSLATIONS_UPDATED',
```

Reply:
updated!